shopinvader_membership/services/membership.py

```diff
@@ -41,21 +41,22 @@ def search(self, **params):
         """
         return self._paginate_search(**params)
 
-    def subscribe(self, _id):
+    def subscribe(self, _id, **params):
```

Comment:
I'm not super familiar with the shopinvader style, but is this use of `**params`  proper ? Where is the input schema declared ?

Reply:
Ok, so after investigation, it looks like this API has a design error since the beginning. The `_id` that is passed here is redundant with the  membership_product_id body. Here the api should not accept any id in the path, and only work with the body parameter to create the new membership invoice.

So what we need is a `POST /membership/subscribe` with membership_product_id in the body, and keep a deprecated `POST /membership/{_id}/subscribe` for backward compatibility.

@acsonefho this is a good exercise to do it properly (using `@rest_api.method`). Let's do this after your holidays.